en/middleware/modules_simulation.md

```diff
@@ -0,0 +1,33 @@
+# Modules Reference: Simulation
```

Comment:
You can remove this too, the docs get created automatically when your module is added (though we will need to check this).

Reply:
@hamishwillee 
what about the link pointing to this file? so far it's pointing to an unexisting file.

Reply:
We won't submit this until the module goes in. At that point the docs should be generated and we will be able to test (or at least validate) the link.